grafana/deployment.yaml

```diff
@@ -0,0 +1,68 @@
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: grafana
+  labels:
+    app: grafana
+spec:
+  selector:
+    matchLabels:
+      app: grafana
+  serviceName: "grafana"
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: grafana
+    spec:
+      priorityClassName: infrastructure
+      securityContext:
+        fsGroup: 472
+        runAsNonRoot: true
+        runAsUser: 472
+      volumes:
+        - name: grafana-configmap-volume
+          configMap:
+            name: grafana-config
+      containers:
+       - name: grafana
+         image: grafana/grafana:latest
```

Comment:
Pick and pin a version, latest version seems fine to me. We prefer pinned version to latest because it gives us reproducibility (modulo people overwriting docker tags). A separate SRE thing we need to address is systematically updating versions.